code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -375,3 +375,117 @@
+	//head
+	if(head)
+		. += "[t_He] [t_is] wearing [head.get_examine_string(user)] on [t_his] head."
+	//suit/armor
+	if(wear_suit)
+		. += "[t_He] [t_is] wearing [wear_suit.get_examine_string(user)]."
+	//back
+	if(back)
+		. += "[t_He] [t_has] [back.get_examine_string(user)] on [t_his] back."
+
+	//Hands
+	for(var/obj/item/I in held_items)
+		if(!(I.item_flags & ABSTRACT))
+			. += "[t_He] [t_is] holding a [weightclass2text(I.w_class)] item in [t_his] [get_held_index_name(get_held_index_of_item(I))]."
+
+	//gloves
+	if(gloves && !(ITEM_SLOT_GLOVES in obscured))
+		. += "[t_He] [t_has] [gloves.get_examine_string(user)] on [t_his] hands."
+
+
+	//handcuffed?
+	if(handcuffed)
+		if(istype(handcuffed, /obj/item/restraints/handcuffs/cable))
+			. += "<span class = 'warning'>[t_He] [t_is] [icon2html(handcuffed, user)] restrained with cable!</span>"
+		else
+			. += "<span class = 'warning'>[t_He] [t_is] [icon2html(handcuffed, user)] handcuffed!</span>"
+
+	//belt
+	if(belt)
+		. += "[t_He] [t_has] [belt.get_examine_string(user)] about [t_his] waist."
+
+	//shoes
+	if(shoes && !(ITEM_SLOT_FEET in obscured))
+		. += "[t_He] [t_is] wearing [shoes.get_examine_string(user)] on [t_his] feet."
+
+	//mask
+	if(wear_mask && !(ITEM_SLOT_MASK in obscured))
+		. += "[t_He] [t_has] [wear_mask.get_examine_string(user)] on [t_his] face."
+
+	if(wear_neck && !(ITEM_SLOT_NECK in obscured))
+		. += "[t_He] [t_is] wearing [wear_neck.get_examine_string(user)] around [t_his] neck."
+
+	//eyes
+	if(!(ITEM_SLOT_EYES in obscured))
+		if(glasses)
+			. += "[t_He] [t_has] [glasses.get_examine_string(user)] covering [t_his] eyes."
+		else if(eye_color == BLOODCULT_EYE && iscultist(src) && HAS_TRAIT(src, CULT_EYES))
+			. += "<span class = 'warning'><B>[t_His] eyes are glowing an unnatural red!</B></span>"
+
+	//ears
+	if(ears && !(ITEM_SLOT_EARS in obscured))
+		. += "[t_He] [t_has] [ears.get_examine_string(user)] on [t_his] ears."
+
+
+	var/list/msg = list("")
+
+	var/list/missing = list(BODY_ZONE_HEAD, BODY_ZONE_CHEST, BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
+	for(var/X in bodyparts)
+		var/obj/item/bodypart/body_part = X
+		missing -= body_part.body_zone
+
+
+	//stores missing limbs
+	var/l_limbs_missing = 0
+	var/r_limbs_missing = 0
+	for(var/t in missing)
+		if(t==BODY_ZONE_HEAD)
+			msg += "<span class='deadsay'><B>[t_His] [parse_zone(t)] is missing!</B><span class='warning'>\n"
+			continue
+		if(t == BODY_ZONE_L_ARM || t == BODY_ZONE_L_LEG)
+			l_limbs_missing++
+		else if(t == BODY_ZONE_R_ARM || t == BODY_ZONE_R_LEG)
+			r_limbs_missing++
+
+		msg += "<B>[capitalize(t_his)] [parse_zone(t)] is missing!</B>\n"
+
+	if(l_limbs_missing >= 2 && r_limbs_missing == 0)
+		msg += "[t_He] look[p_s()] all right now.\n"
+	else if(l_limbs_missing == 0 && r_limbs_missing >= 2)
+		msg += "[t_He] really keeps to the left.\n"
+	else if(l_limbs_missing >= 2 && r_limbs_missing >= 2)
+		msg += "[t_He] [p_do()]n't seem all there.\n"
+
```

Comment:
Pretty sure this doesn't handle better limbs 
@Kapu1178

Reply:
Me no like. This completely undoes the changes I made to examine for Kapulimbs